app/models/employee.rb

```diff
@@ -6,6 +6,10 @@ class Employee < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  has_many :given_kudos, class_name: 'Kudo', foreign_key: 'giver_id', dependent: :destroy, inverse_of: :kudo
-  has_many :received_kudos, class_name: 'Kudo', foreign_key: 'receiver_id', dependent: :destroy, inverse_of: :kudo
+  has_many :given_kudos, class_name: 'Kudo', foreign_key: 'giver_id', dependent: :destroy, inverse_of: :giver
+  has_many :received_kudos, class_name: 'Kudo', foreign_key: 'receiver_id', dependent: :destroy, inverse_of: :receiver
+
+  def kudo_points
+    received_kudos.count
```

Comment:
I would consider adding here some safeguad - someting like:
`return 0 if reveived_kudos.blank`

Reply:
I wonder if it is needed because:
1. Each kudo must have a received employee
2. If there are no kudos, it returns 0

Is there any reason to use this?

Reply:
the safeguard is already being handled by AR's `count` method